src/index.js

```diff
@@ -5,10 +5,11 @@ import './index.scss'
 import { Layout } from './components/layout'
 import { BrowserRouter as Router } from 'react-router-dom'
 
-const App = () => (
-  <Router>
+const App = () => {
+  window.$password = 'luxoft';
+  return (<Router>
     <Layout />
-  </Router>
-)
+  </Router >);
```

Comment:
No space needed here. Remove for legibility / cleaner diffs

Reply:
Done